query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/converters/PinotFunctionConverterConfig.java

```diff
@@ -0,0 +1,38 @@
+package org.hypertrace.core.query.service.pinot.converters;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@Value
+@AllArgsConstructor
+public class PinotFunctionConverterConfig {
+
+  private static final String PERCENTILE_AGGREGATION_FUNCTION_CONFIG = "percentileAggFunction";
+  private static final String DISTINCT_COUNT_AGGREGATION_FUNCTION_CONFIG =
+      "distinctCountAggFunction";
+  private static final String DEFAULT_PERCENTILE_AGGREGATION_FUNCTION = "PERCENTILETDIGEST";
+  private static final String DEFAULT_DISTINCT_COUNT_AGGREGATION_FUNCTION = "DISTINCTCOUNT";
```

Comment:
I think we should make DISTINCT_COUNT_HLL as the default.

Reply:
I left it this way for backwards compatibility. Without any custom config, it'll retain identical behavior as before, but it's easily changed.